src/Microsoft.ML.CpuMath/SseIntrinsics.cs

```diff
@@ -221,7 +221,7 @@ public static unsafe void MatMul(float[] mat, float[] src, float[] dst, int crow
                             length -= misalignment;
                         }
 
-                        if (length > 4)
+                        if (length > 3)
```

Comment:
>> 3 [](start = 35, length = 3)

Not sure about performance, but readability wise, i would change it to >=4 instead of >3.

Reply:
cc @tannergooding

Reply:
I'm not sure I agree from a readability standpoint, most code I see operates in terms of `<` and `>` by default and `length > 3` is generally easy to read/understand.